app/src/main/java/net/squanchy/typeface/TypefaceDelegate.kt

```diff
@@ -0,0 +1,32 @@
+package net.squanchy.typeface
+
+import android.graphics.Paint
+import android.graphics.Typeface
+import net.squanchy.extensions.whenTrue
+
+class TypefaceDelegate(private val newTypeface: Typeface) {
+
+    private companion object {
+        @SuppressWarnings("MagicNumber")
+        const val TEXT_SKEW_X = -.25f
+        const val FALSE_FLAG = 0
+    }
+
+
+    fun applyStyleTo(paint: Paint) {
```

Comment:
what does this do?

Reply:
maybe we could rename it `applyTypefaceTo`?

Reply:
Sounds good